core/src/main/java/net/sourceforge/jnlp/cache/ResourceDownloader.java

```diff
@@ -403,7 +378,7 @@ private void downloadPackGzFile(URLConnection connection, URL downloadFrom, URL
         markForDelete(downloadFrom);
     }
 
-    private void downloadGZipFile(URLConnection connection, URL downloadFrom, URL downloadTo) throws IOException {
+    private void downloadGZipFile(CloseableConnection connection, URL downloadFrom, URL downloadTo) throws IOException {
```

Comment:
final params

Reply:
will do all the parameters in a later PR